libs/design/src/molecules/modal/modal/modal.component.spec.ts

```diff
@@ -104,22 +98,31 @@ describe('DaffModalComponent', () => {
     });
   });
 
-  describe('on <backdrop>', () => {
+  describe('on <daff-backdrop>', () => {
 
     beforeEach(() => {
       backdrop = fixture.debugElement.query(By.css('daff-backdrop')).componentInstance;
     });
     
-    it('should set show to modal.show', () => {
-      expect(backdrop.show).toEqual(modal.show);
+    it('should remove add and remove the backdrop from the DOM as show changes', () => {
```

Comment:
I think you have an extra "remove" here

Reply:
Fixed in 27f8d1cab60c6f637d17d13cbf78d8072b565b8f